mxcubeweb/core/models/generic.py

```diff
@@ -11,3 +11,20 @@ class SimpleNameValue(BaseModel):
 class AppSettingsModel(BaseModel):
     mode: ModeEnum = Field(ModeEnum.OSC, description="MXCuBE mode SSX or OSC")
     version: str = Field("", description="MXCuBE version")
+    mesh_result_format: str = Field(
+        "PNG", description="Format of mesh result for display"
+    )
+    use_native_mesh: bool = Field(
+        True, description=" Use the native mesh feature available, true by default"
```

Comment:
please remind me what this one meant... this would be the md3 command? no workflow?

Reply:
Exactly, so its basically just to disable the native mesh method so that its not available to the users. Its useful for instance for us as we have the workflow meshes, and it would mean that there are two kinds of meshes and that is of course a bit confusing. So this option is simply to disable the native one and favor the workflow one.

Reply:
good, less confusion is always good :)